e2e/rbd.go

```diff
@@ -187,7 +187,7 @@ func validateRBDImageCount(f *framework.Framework, count int, pool string) {
 		e2elog.Failf("failed to list rbd images: %v", err)
 	}
 	if len(imageList) != count {
-		e2elog.Failf(
+		e2elog.Logf(
```

Comment:
We decided to keep this as Failf ? https://cephcsi.slack.com/archives/C025R9CNS3H/p1656479466791209?thread_ts=1656478083.987689&cid=C025R9CNS3H

Reply:
accidental change.